objects/src/accounts/package/storage_entry/mod.rs

```diff
@@ -376,29 +439,44 @@ impl MapEntry {
     pub fn new(key: impl Into<WordRepresentation>, value: impl Into<WordRepresentation>) -> Self {
         Self { key: key.into(), value: value.into() }
     }
+
+    pub fn template_keys(&self) -> impl Iterator<Item = &TemplateKey> {
+        self.key.template_keys().chain(self.value.template_keys())
+    }
+
+    pub fn into_parts(self) -> (WordRepresentation, WordRepresentation) {
+        let MapEntry { key, value } = self;
+        (key, value)
+    }
 }
 
 #[cfg(test)]
```

Comment:
nit: let's add a section separator above this line.

Reply:
Added